### PR TITLE
porn与hub字体大小分离设置

### PR DESCRIPTION
**Pull request by kentwood**

### CODE REVIEW

File: src/generator/Pornhub.vue [hunk 1]
```diff
@@ -84,13 +84,13 @@
...
         id="logo"
-        :style="{ 'font-size': fontSize + 'px', 'background-color': transparentBgColor,'font-family':fontFamily }"
+        :style="{ 'background-color': transparentBgColor,'font-family':fontFamily }"
       >
         <template v-if="!reverseHighlight">
           <span
             @input="updatePrefix"
             class="prefix"
-            :style="{ color: prefixColor }"
+            :style="{ 'font-size': pornFontSize + 'px', color: prefixColor }"
```
Comment (bestony) [inline src/generator/Pornhub.vue:93]: use `prefixFontSize` replace `pornFontSize`

File: src/generator/Pornhub.vue [hunk 2]
```diff
@@ -100,7 +100,7 @@
           </span>
           <span
             class="postfix"
-            :style="{ color: suffixColor, 'background-color': postfixBgColor }"
+            :style="{ 'font-size': hubFontSize + 'px', color: suffixColor, 'background-color': postfixBgColor }"
```
Comment (bestony) [inline src/generator/Pornhub.vue:103]: use `suffixFontSize` replace `hubFontSize`

**Review by bestony — CHANGES_REQUESTED** [review]

File: src/generator/Pornhub.vue (hunk 1 above)
Comment (Janusz290) [reply src/generator/Pornhub.vue:93]: 6

**Review by Janusz290 — APPROVED** [review]

**Review by Janusz290 — APPROVED** [review]

**Review by Janusz290 — APPROVED** [review]
Z

**Review by Janusz290 — APPROVED** [review]

**Pull request closed without merging**